sips/sip-015/sip-015-network-upgrade.md

```diff
@@ -0,0 +1,1582 @@
+they had locked them for subsequent reward cycles.
+
+If a user has amassed more STX, or believes that they can acquire a reward slot
+in a subsequent cycle, then they can re-stack their STX.
+
+The current system behavior is to keep the user's STX locked for the duration of
+their specified lock-up period, even if they are not earning them any reward
+slots.
+
+### Changed: Automatic PoX Sunset Removed
+
+This SIP proposes removing the automatic PoX sunset that is currently slated to
+activate in the current system.  The rationale for the PoX sunset was to address
+an incentive problem with PoX: if miners acquire enough STX, they can mine at a
+discount because the burnchain tokens they pay will be paid back to themselves.
+While the existence of Stacking pools means this can't be avoided at any scale,
+it is particularly bad for the chain if miners have so many STX Stacked that
+they can control the _median_ mining commitment value in the sortition weight
+calculation.  If they can do this, then they can spend as many burnchain tokens
+as they want since they will get them back right away.  The effect of this
+behavior is that miners who can discount-mine will eventually price out all
+other miners, leading to a chain where only large STX-holders can effectively mine.
+
+The PoX sunset fixes this incentive problem by capping any gains such a miner
+could ever make over the system's lifetime.  Even if a miner did this, it would
+lead to a temporary gain in their mining power.  However, Stacking has proven
+successful and popular, and we believe that stopping its operation on an
+automated schedule has the potential to harm the Stacks blockchain ecosystem.
+
+As an alternative, we recommend that discount-mining behavior be policed by
+vigilent users.  In the system today, 25% of the liquid STX can vote to stop PoX
+payouts for the next reward cycle.  If discount-mining behavior becomes the
+dominant strategy in Stacks, then users already have the power to fix the
+miner incentives _if it becomes a problem_.  We do not know when, or if, it ever
+will, but we believe that the mechanism(s) for determining when or if PoX
+deactivates to counter discount-mining must be (1) adaptive in the face of an
+ever-shifting set of discount-mining strategies and the strategies for
+countering them, and (2) under the control of users, not Stacks blockchain developers.
+The PoX sunset has neither of these properties.
+
+A future SIP may propose an alternative mechanism for empowering users to
+collectively address discount-mining in the event that either
+a better strategy be discovered, the Stacks blockchain incentives
+get altered to render this impractical.
+
+### Changed: Support Segwit PoX Payout Addresses
+
+The type of a PoX address is now `(tuple (hashbytes (buff 32)) (version (buff
+1)))`.  This is to accomodate p2wsh and p2tr scriptPubKeys on Bitcoin.  In
+addition, new values for `version` are supported to represent these encodings:
```

Comment:
Do new version values also imply new stx addresses? Do we need to update this [PR  for CAIP-10](https://github.com/ChainAgnostic/namespaces/pull/27/files#r1002478490)?

Reply:
I don't think so.  These only pertain to PoX addresses, which are all Bitcoin addresses.  Stacks addresses are not changing in this SIP.